website/static/js/contribAdder.js

```diff
@@ -336,8 +319,11 @@ var AddContributorViewModel = oop.extend(Paginator, {
         $osf.block();
         return $osf.postJSON(
             nodeApiUrl + 'contributors/', {
-                users: self.selection().map(function(user) {
-                    return ko.toJS(user);
+                users: ko.utils.arrayMap(self.selection(), function(user) {
+                    var permission = user.permission().value; //removes the value from the object
+                    var tUser = JSON.parse(ko.toJSON(user)); //The serialized user minus functions
```

Comment:
@sloria 

https://github.com/knockout/knockout/issues/251

For TL;DR: Basically toJS keeps the functions, toJSON removes ko functions and an error associated with them. They talked about removing functions with toJS, but decided against it as there are people that rely on the return of functions.